src/app/services/product.service.ts

```diff
@@ -129,32 +106,16 @@ export class ProductService {
     }
   }
 
-  cartCount(count: number): void {
-    this.productCount += count;
-    localStorage.setItem(
-      this.productCountKey,
-      JSON.stringify(this.productCount)
-    );
-  }
-
-  getCartProducts(): ProductData[] {
+  public getCartProducts(): ProductData[] {
     return this.cartProducts;
   }
 
-  removeProductFromCart(product: ProductData): void {
-    const index = this.cartProducts.findIndex((p) => p.id === product.id);
+  public removeProductFromCart(product: ProductData): void {
+    const index = this.cartProducts.findIndex(p => p.id === product.id);
```

Comment:
```suggestion
    const index = this.cartProducts.findIndex({id} => id === product.id);
```

Reply:
the store is being used so this code has been removed

Reply:
the store is being used so this code has been moved to a util file